<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelSelectPage</class>
 <widget class="QWidget" name="LevelSelectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="level9_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level 9</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">levelButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="level4_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level 4</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">levelButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="level1_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level 1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">levelButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="level2_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level 2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">levelButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="level7_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level 7</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">levelButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="level10_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level 10</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">levelButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="level5_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level 5</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">levelButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="level3_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level 3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">levelButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="level8_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level 8</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">levelButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>540</y>
     <width>72</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level6_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level 6</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">levelButtons</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="levelButtons"/>
 </buttongroups>
</ui>
